lib/alarmist.ex

```diff
@@ -75,13 +75,16 @@ defmodule Alarmist do
   end
 
   @doc """
-  Return all of the currently set alarms
+  Return all of the currently set alarms and their descriptions
   """
-  @spec current_alarms() :: [alarm_id()]
+  @spec current_alarms() :: [{alarm_id(), any()}]
```

Comment:
@doawoo briefly asked about maps. I don't think there's a use case for a map. I.e., if you are interested in a specific alarm, you should be listening for it rather than polling, and if you insist on polling it would be better to have a function to just get the alarm rather than returning a long list/map of random stuff you don't care about.

I agree that the spec should be:

```suggestion
  @spec current_alarms() :: [alarm()]
```

That actually reads better to me. I.e., `current_alarms/0` returns `alarm`'s rather than `alarm_id`'s.

Reply:
I'll update the types and get this ready to merge!